ci-operator/jobs/openshift/release/openshift-release-release-4.3-periodics.yaml

```diff
@@ -6527,6 +6527,259 @@ periodics:
     - name: release-pull-secret
       secret:
         secretName: ci-pull-credentials
+- agent: kubernetes
+  cluster: api.ci
+  decorate: true
+  interval: 12h
+  labels:
+    ci.openshift.io/release-type: informing
+    job-env: remote-libvirt
+    job-release: "4.3"
+    job-test: e2e
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: release-openshift-origin-installer-e2e-remote-libvirt-ppc64le-4.3
+  spec:
+    containers:
+    - args:
+      - --artifact-dir=$(ARTIFACTS)
+      - --give-pr-author-access-to-namespace=true
+      - --image-import-pull-secret=/etc/pull-secret/.dockerconfigjson
+      - --input-hash=$(BUILD_ID)
+      - --input-hash=$(JOB_NAME)
+      - --kubeconfig=/etc/apici/kubeconfig
+      - --lease-server-password-file=/etc/boskos/password
+      - --lease-server-username=ci
+      - --lease-server=https://boskos-ci.svc.ci.openshift.org
+      - --secret-dir=/usr/local/e2e-remote-libvirt-cluster-profile
+      - --secret-dir=/usr/local/pull-secret
+      - --target=e2e-remote-libvirt
+      - --template=/usr/local/e2e-remote-libvirt
+      command:
+      - ci-operator
+      env:
+      - name: RELEASE_IMAGE_LATEST
+        value: registry.svc.ci.openshift.org/ocp/release:4.3-ci
+      - name: BRANCH
+        value: "4.3"
+      - name: ARCH
+        value: ppc64le
+      - name: CLUSTER_TYPE
+        value: libvirt-ppc64le
+      - name: BASTION_SVC
+        value: sshd
+      - name: BASTION_SVC_IP
+        value: 172.30.168.41
+      - name: REMOTE_LIBVIRT_HOSTNAME
+        value: c155f2u33
+      - name: MULTIARCH_RELEASE_IMAGE_LATEST
+        value: registry.svc.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.3.0-0.nightly-ppc64le-2020-04-07-135751
+      - name: CONFIG_SPEC
+        value: "resources:\n  '*':\n    limits:\n      memory: 4Gi\n    requests:\n
```

Comment:
Is this something we can reformat so that it looks more like the 4.2 libvirt spec?
It's a little hard to read this way.

Reply:
i did that b/c the prow job yelled at me. i agree. if you want i can undo it and you can merge over its wishes. but that's why it's hard to read :)

Reply:
We can leave it :)